msteams-platform/bots/bot-features.md

```diff
@@ -47,6 +48,105 @@ The [Bot Framework](https://dev.botframework.com/) is a rich SDK used to create
 
 Webhooks and connectors connect your bot to your web services. Using webhooks and connectors, you can create a bot for basic interaction, such as creating a workflow or other simple commands. They are available only in the team where you create them and are intended for simple processes specific to your company's workflow. For more information, see [what are webhooks and connectors](~/webhooks-and-connectors/what-are-webhooks-and-connectors.md).
 
+## Azure bot service
+
+The Azure bot service, along with the Bot Framework, provides tools to build, test, deploy, and manage intelligent bots, all in one place. You can also create your bot in Azure bot service.
+
+> [!IMPORTANT]
+> Bot applications within Microsoft Teams are available in GCC-High through [Azure bot Service](/azure/bot-service/channel-connect-teams).
+
+> [!NOTE]
+> * Bots in GCCH only support  up to manifest version v1.10.
+> * Image URL's in Adaptive Cards are not supported in GCCH environment. You can replace an image URL with Base64 encoded DataUri.
+> * Bot channel registration in Azure Government will provision web app bot, app service (app service plan), and application insights also but it doesn't support to provision the azure bot service only (no app service).
+>   <details>
+>   <summary><b>If you want to do bot registration only</b></summary>
+>
+>   * Go to the resource group and manually delete the unused resources. Such as the app service, app service plan (if you created during bot registration), and the application insights (if you choose to enable it during bot registration).
+>   * You can also use az-cli to do bot registration:
+>
+>     1. Sign into azure and set the subscription <br> 
+>           &nbsp; az cloud set –name  "AzureUSGovernment" <br> 
+>           &nbsp; az account set –name "`subscriptionname/id`".<br>
+>     1. Create app registration  
+>           &nbsp; az ad app create --display-name "`name`" <br> 
+>           &nbsp; --password "`password`" --available-to-other-tenants.<br> 
+>           Your app id would be created here.<br>
+>     1. Create bot resource <br>
+>           &nbsp; az bot create –resource-group "`resource-group`"<br>
+>           &nbsp; --appid "`appid`"<br>
+>           &nbsp; --name "`botid`"<br>
+>           &nbsp; --kind "registration".<br>
+>
+> </details>
+
+For GCCH environment, you need to register a bot using [Azure Government portal](https://portal.azure.us).
```

Comment:
Let's explicitly state what is currently implied, which is that bots registered through Azure commercial subscription, or Dev Center/App Studio, will not be available in GCCH.

Reply:
Also, this means that an Azure Government subscription (let's use the proper name) is required, correct?